app/uk/gov/hmrc/emailverification/repositories/JourneyRepository.scala

```diff
@@ -52,6 +53,7 @@ private class JourneyMongoRepository @Inject() (mongoComponent: ReactiveMongoCom
     idFormat       = implicitly) with JourneyRepository {
 
   override def indexes: Seq[Index] = Seq(
+    Index(key    = Seq("journeyId" -> IndexType.Ascending), name = Some("journeyId"), unique = true),
     Index(key     = Seq("createdAt" -> IndexType.Ascending), name = Some("ttl"), options = BSONDocument("expireAfterSeconds" -> 4.hours.toSeconds))
```

Comment:
similar to `verificationStatusRepositoryTtl` in `application.conf`, it's better to have this `expireAfterSeconds` configurable.

Reply:
journeyId is not date value and i can't set ttl. Index was missing for journeyId so it is added here.